File: Cargo/Tags/Tags.m
//
//  Tags.m
//  Cargo
//
//  Created by Julien Gil on 06/03/2017.
//  Copyright © 2017 François K. All rights reserved.
//

#import "Tags.h"
#import "Cargo-Swift.h"

@implementation Tags

- (NSObject*)executeWithParameters:(NSDictionary*)parameters {
    [[Cargo getInstance] execute:parameters];
    [CargoItem notifyTagFired];
    return nil;
}

@end

File: Cargo/Tags.swift
//
//  Tags.swift
//  Cargo
//
//  Created by Julien Gil on 20/01/2017.
//  Copyright © 2017 François K. All rights reserved.
//

import Foundation
import GoogleTagManager

final class Tags: NSObject, TAGCustomFunction {
    
    func execute(withParameters parameters: [AnyHashable : Any]!) -> NSObject! {
        Cargo.getInstance().execute(parameters);
        return nil;
    }
}
